ui_dashboard_workbook/arc_windows_esu/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+type: docs
+title: "Azure Arc Windows ESU Dashboard"
+linkTitle: "Azure Arc Windows ESU Dashboard"
+weight: 1
+description: >
+---
+
+## Azure Arc Windows ESU Dashboard
+
+This dashboard has been created to enable you to view several elements of the Windows or Linux servers you have deployed the Azure Arc agent to in one simple view.   Within this dashboard you will be able to see: 
+* The Windows/Linux servers that have the Arc agent installed
+* A count of what operating systems are being used by these Arc enabled servers
+* A view of what Arc agent version is installed
+* A count of any SQL instances that have been detected on the servers with Arc agents installed
+* Current Windows ESU assignment status
+* An overview of whether the Azure Arc is allowed to enable [extensions](https://learn.microsoft.com/azure/azure-arc/servers/manage-vm-extensions) or not.  And the status of some of the extensions if they are allowed to be used. 
+
+## Contributors
+
+This Jumpstart Drop was originally written by the following contributors:
+
+- [Sarah Lean | Senior Technical Specialist at Microsoft](www.linkedin.com/in/lauranicolasd)
+
+## Prerequisites
+
+- Clone the Azure Arc Drops repository
+
+    ```shell
+    git clone https://github.com/Azure/arc_jumpstart_drops.git
+    ```
+
+## Getting Started
+
+### Import the Azure Monitor Workbook
+
+To import the workbook navigate to the [workbook directory](https://github.com/Azure/arc_jumpstart_drops/workbooks/arc_windows_esu)and run these Azure CLI commands:
+
```

Comment:
What about for user running Windows? Bash is only for Linux, maybe also have the Windows version?